src/rotten-tomatoes/SpoilTomatoes.js

```diff
@@ -0,0 +1,71 @@
+class SpoilTomatoes {
+
+    constructor(grid) {
+        this.grid = grid;
+        this.colLength = 0;
+        this.rowLength = 0;
```

Comment:
You can initialize these here. Assuming the column length is the same for all rows.

Reply:
Okay